lib/PuppeteerSharp.Tests/BrowserContextTests/DefaultBrowserContextTests.cs

```diff
@@ -52,7 +52,7 @@ public async Task PageSetCookiesAsyncShouldWork()
                 Value = "John Doe"
             });
 
-            var cookie = (await Page.GetCookiesAsync()).FirstOrDefault();
+            var cookie = (await Page.GetCookiesAsync()).First();
```

Comment:
no null check after a `FirstOrDefault` implies it can never be empty, and hence should be a `First`